index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Porfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;300;500;600;700;800&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="./assets/style/style.css">
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="nav-container">
+            <div class="nav-text">Welcome</div>
+            <div class="hamburgur-menu">
+                <img class="hamburger" src="./assets/icons/hamburger.png" alt="menu-hamburger">
+            </div>
+        </div>
+    </nav>
+
+    <div class="headline-container">
+
+        <div class="headline">
+            <div class="primary-headline">
+                <p class="primary-headline-text">Hey there. I’m Ansderly. <br>
+                    <span class="primary-headline-text-second-color"> I’m a software developer</span>
+                </p>
```

Comment:
- [ ] Please kindly ensure that the main title is an `h1` tag. If you want that google (and other search engines) know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.